gulpfile.js

```diff
@@ -42,6 +42,19 @@ function icons() {
     pipe(dest("./html/images/icons/dist"));
 }
 
+function attributes_icons() {
```

Comment:
`_` somewhat breaks with the naming convention of all other methods in this file. Maybe use `attributeIcons` instead?

Reply:
sure, I'll change it.

Reply:
changed.